### ISSUE TITLE
Make F14/F15 enabled by default

### ISSUE THREAD

**Issue by skyynet** [MonitorControl/MonitorControl#623]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

After upgrading from 3.0.0 to 3.1.0 I experienced brightness control no longer working. After enabling the F14/F15 keys it can be done by those AND the dedicated keys. Using LG monitor & Logitech MX Keys keyboard.

### Steps to reproduce

click brightness down/up

### Expected behavior

brightness de-/increase

### Environment Information (please complete the following information)

```markdown
- macOS version: 11.6
- Mac model: MacBook Pro (13-inch, 2016, 4 Thunderbolt 3 ports)
- MonitorControl version: 3.1.0
- Monitor(s): LG 34UC99
- Apple Silicon/M1 (yes or no): no
```

**Comment by JoniVR** [comment]
If I understand this correctly, you're saying that the default brightness keys weren't working at all before you toggled the F14/F15 setting, and after enabling it, both options worked (as intended)?

**Comment by JoniVR** [comment]
If this is the case, then the MX Keys might possibly be mapping brightness to F14/F15 keycodes instead of regular brightness keycodes would be my first guess.

**Comment by waydabber** [comment]
In 4.0.0 I reversed how this setting works, F14/F15 is enabled by default.

**Comment by JoniVR** [comment]
We should not be waiting for 4.0.0 if this is an issue in 3.1.0

**Comment by JoniVR** [comment]
In general I think we should move away from monster releases, a lot of features can be done separately in v3.x.x releases and make little sense in a bigger release imo. I'll create a fix for 3.1.1

**Comment by skyynet** [comment]
> If I understand this correctly, you're saying that the default brightness keys weren't working at all before you toggled the F14/F15 setting, and after enabling it, both options worked (as intended)?

Yes, that'S true. As there was no F14/F15 mapping in 3.0.0 and it worked for me there, I assume something changed. Since the F14/F15 keys are pretty hard to access by mistake and I'm not aware of any software using it by default, enabling them by default sounds like a good workaround to me. Then Monitor Control works instantly for the user.
